### ISSUE TITLE
How I can fix this error: Unable to find domain in "localhost:8080"

### ISSUE THREAD

**Issue by duhon** [sitespeedio/sitespeed.io#2687]
I'm runing a website through a docker with URL http://localhost:8080/build
Then run test ``docker run --network host --rm -v $(pwd)/sitespeed:/sitespeed.io sitespeedio/sitespeed.io:10.1.1 --config config.json http://localhost:8080/build``
I get this error:
```Google Chrome 77.0.3865.75
Mozilla Firefox 69.0
INFO: Versions OS: linux 4.4.0-72-generic nodejs: v10.16.0 sitespeed.io: 10.1.1 browsertime: 6.1.1 coach: 4.0.1
INFO: Running tests using Chrome - 1 iteration(s)
INFO: Navigating to url http://localhost:8080/build/login iteration 1
INFO: Testing url http://localhost:8080/build iteration 1
INFO: http://localhost:8080/builds 21 requests, backEndTime: 765ms, firstPaint: 885ms, firstVisualChange: 500ms, DOMContentLoaded: 994ms, Load: 1.10s, speedIndex: 898, visualComplete85: 900ms, lastVisualChange: 2.83s, rumSpeedIndex: 885
ERROR: Error: Unable to find domain in "localhost:8080"
   at getDomainFromOriginOrURL (/usr/src/app/node_modules/third-party-web/lib/create-entity-finder-api.js:9:9)
   at getRootDomain (/usr/src/app/node_modules/third-party-web/lib/create-entity-finder-api.js:13:18)
   at getEntityInDataset (/usr/src/app/node_modules/third-party-web/lib/create-entity-finder-api.js:21:22)
   at Object.module.exports.getThirdParty (/usr/src/app/node_modules/webcoach/lib/har/thirdParty.js:11:19)
   at Object.processPage (/usr/src/app/node_modules/webcoach/lib/har/privacy/thirdPartyPrivacy.js:14:37)
   at Object.module.exports.runAdvice (/usr/src/app/node_modules/webcoach/lib/har/index.js:24:31)
   at Object.runHarAdvice (/usr/src/app/node_modules/webcoach/lib/index.js:119:22)
INFO: HTML stored in /sitespeed.io/sitespeed-result/temp_data_for_first_run```


I use Ubuntu 14 for base host

**Comment by soulgalore** [comment]
Hi @duhon checkout https://www.sitespeed.io/documentation/sitespeed.io/docker/#access-localhost 

Best
Peter

**Comment by duhon** [comment]
@soulgalore yes, I'm using ``--network=host`` this helps, but the thirdParty module wants to get data from a non-existent URL http://localhost:8080/

**Comment by soulgalore** [comment]
Aha I see. Let me fix that for the next release.

**Comment by soulgalore** [comment]
This is fixed in 10.2.0 just released, please verify that it works for you when you have time. The problem is upstream in the Third party lib, I'll create an upstream issue when I find the time.

**Comment by duhon** [comment]
I have checked. In my case, everything works well. Thanks for the quick help
